src/libraries/ErrorsLib.sol

```diff
@@ -7,22 +7,19 @@ pragma solidity ^0.8.0;
 /// @notice Library exposing error messages.
 library ErrorsLib {
     /// @notice Thrown when the caller is not the owner nor an updater.
-    string internal constant CALLER_NOT_OWNER_OR_UPDATER = "caller is not the owner or updater";
+    string internal constant CALLER_NOT_OWNER_NOR_UPDATER = "caller is not the owner nor an updater";
```

Comment:
```suggestion
    string internal constant CALLER_NOT_OWNER_NOR_UPDATER = "not owner nor updater";
```

Reply:
we usually don't write error sentences but error keywords (in Blue, MM, and other private projects)